en/tidb-toolkit.md

```diff
@@ -178,87 +247,121 @@ tidb-ctl schema in mysql
 
-```shell
-helm search pingcap -l
-```
+    - If the Helm version < 2.16.0:
 
-If the Helm version >= 2.16.0:
+        {{< copyable "shell-regular" >}}
 
-{{< copyable "shell-regular" >}}
+        ```shell
+        helm search pingcap -l
+        ```
 
-```shell
-helm search pingcap -l --devel
-```
+    - If the Helm version >= 2.16.0:
 
-```
-NAME                    CHART VERSION   APP VERSION DESCRIPTION
-pingcap/tidb-backup     v1.0.0                      A Helm chart for TiDB Backup or Restore
-pingcap/tidb-cluster    v1.0.0                      A Helm chart for TiDB Cluster
-pingcap/tidb-operator   v1.0.0                      tidb-operator Helm chart for Kubernetes
-...
-```
+        {{< copyable "shell-regular" >}}
 
-When a new version of chart has been released, you can use `helm repo update` to update the repository cached locally:
+        ```shell
+        helm search pingcap -l --devel
+        ```
 
-{{< copyable "shell-regular" >}}
+    ```
+    NAME                    CHART VERSION   APP VERSION DESCRIPTION
+    pingcap/tidb-backup     v1.0.0                      A Helm chart for TiDB Backup or Restore
+    pingcap/tidb-cluster    v1.0.0                      A Helm chart for TiDB Cluster
+    pingcap/tidb-operator   v1.0.0                      tidb-operator Helm chart for Kubernetes
+    ...
+    ```
 
-```shell
-helm repo update
-```
+    When a new version of chart has been released, you can use `helm repo update` to update the repository cached locally:
 
-Common Helm operations include `helm install`, `helm upgrade`, and `helm del`. Helm chart usually contains many configurable parameters which could be tedious to configure manually. For convenience, it is recommended that you configure using a YAML file. Based on the conventions in the Helm community, the YAML file used for Helm configuration is named `values.yaml` in this document.
+    {{< copyable "shell-regular" >}}
 
-When performing a deployment or upgrade, you must specify the chart name (`chart-name`) and the name for the deployed application (`release-name`). You can also specify one or multiple `values.yaml` files to configure charts. In addition, you can use `chart-version` to specify the chart version (by default the latest GA is used). The steps in command line are as follows:
+    ```shell
+    helm repo update
+    ```
 
-* Install:
+    Common Helm operations include `helm install`, `helm upgrade`, `helm del`, and `helm ls`. The Helm chart usually contains many configurable parameters which could be tedious to configure manually. For convenience, it is recommended that you configure these parameters using a YAML file. Based on the conventions in the Helm community, the YAML file used for Helm configuration is named `values.yaml` in this document.
+    Before the operations of installation, upgrade, or deletion, etc., you can execute `helm ls` to view the applications that have been installed in the cluster:
```

Comment:
```suggestion

    Before the operations of installation, upgrade, deletion, and so on, you can execute `helm ls` to view the applications that have been installed in the cluster:
```
Avoid the use of Latin words such as `etc`, `i.e.`, and `e.g.`.

Reply:
OK, thanks!